frontend/src/v5/ui/components/viewer/cards/cardFilters/filterForm/filterFormValues/filterFormValues.component.tsx

```diff
@@ -94,6 +106,13 @@ export const FilterFormValues = ({ type }: { type: CardFilterType }) => {
 			</>
 		);
 	}
+	if (isSelectType(type)) {
+		return (
+			<FormSelect name="values.0.value" formError={!!error?.[0]}>
```

Comment:
In Date filter input, Santiago told me we shall display errors' helperText. I presume the same will be here, so `formError={!!error?.[0]}` should probably become `formError={error?.[0]?.value}` (although, I doubt a select will show any errors)